pkg/network/dns/stats.go

```diff
@@ -177,10 +176,9 @@ func (d *dnsStatKeeper) GetAndResetAllStats() StatsByKeyByNameByType {
 	ret := d.stats // No deep copy needed since `d.stats` gets reset
 	d.stats = make(StatsByKeyByNameByType)
 	droppedStats := statsTelemetry.droppedStats.Load()
-	log.Debugf("[DNS Stats] Number of processed stats: %d, Number of dropped stats: %d", d.processedStats-d.lastProcessedStats, droppedStats-d.lastDroppedStats)
-	d.lastProcessedStats = d.processedStats
-	d.lastDroppedStats = droppedStats
+	log.Debugf("[DNS Stats] Number of processed stats: %d, Number of dropped stats: %d", d.processedStats, droppedStats-d.lastDroppedStats)
 	d.processedStats = 0
+	d.lastDroppedStats = droppedStats
```

Comment:
Do we want to use the same style for `processedStats` and `droppedStats` ? I.e.  both are reset of 0 every time the stats are emitted instead of one being a monotonic counter, and the other not.

Reply:
yeah that works too. it's ultimately the same number of variables